jetty/pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Like I wrote in the README, this file is needed in order to create a simple server that will load our servlets and run them.

Reply:
It was adapted from [here](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/appengine-java11/appengine-simple-jetty-main/pom.xml).